File: code_test/programmers/두개뽑아서더하기.js
function solution(numbers) {
  var answer = [];
  let n = numbers.length;
  for (let i = 0; i < n - 1; i++) {
    for (let j = i + 1; j < n; j++) {
      let sum = numbers[i] + numbers[j];
      if (!answer.includes(sum)) {
        answer.push(sum);
      }
    }
  }
  return answer.sort((a, b) => a - b);
}

File: code_test/programmers/두개뽑아서더하기.py
def solution(numbers):
    answer = []
    n = len(numbers)
    for i in range(n - 1):
        for j in range(i+1, n):
            sum = numbers[i] + numbers[j]
            if sum not in answer:
                answer.append(sum)
    return sorted(answer)
